pyproject.toml

```
@@ -77,8 +77,8 @@ flatpak_sdk = "org.gnome.Sdk"
 
 [tool.briefcase.app.testbed.windows]
 requires = [
-    # Python.net isn't avaialble for 3.13 yet.
-    "pythonnet>=3.0.0rc6; python_version < '3.13'",
+    # Python.net isn't avaialble for 3.14 yet.
+    "pythonnet>=3.0.0rc6; python_version < '3.14'",
```

No need for pre-releases anymore:

```suggestion
    "pythonnet>=3.0.0; python_version < '3.14'",
```